client/my-sites/invites/invite-form-header/index.jsx

```diff
@@ -20,6 +20,12 @@ export default React.createClass( {
 		);
 	},
 
+	getSiteName() {
+		const { site } = this.props;
+
+		return site.title || '';
+	},
```

Comment:
That function was an already existing function, which was then mistakenly removed. :disappointed: 

So, you'll see that the function is getting called in a few places throughout the same file.

We return `''` if a site title doesn't exist because we are using the title in translations and don't want an object.